src/test/java/seedu/finance/logic/parser/SearchCommandParserTest.java

```diff
@@ -27,32 +31,46 @@ public void parse_validNameArgs_returnsSearchCommand() {
         // no leading and trailing whitespaces
         SearchCommand expectedFindCommand =
                 new SearchCommand(new NameContainsKeywordsPredicate(Arrays.asList("Apple", "Banana")));
-        assertParseSuccess(parser, "-name Apple Banana", expectedFindCommand);
```

Comment:
Good job using variables instead of typing the flag out as it 
allows reuse-ability